src/main/java/frc/robot/subsystems/intake/pivot/PivotCommandBuilder.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.intake.pivot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+
+public class PivotCommandBuilder {
+
+	private final Pivot pivot;
+
+	public PivotCommandBuilder(Pivot pivot) {
+		this.pivot = pivot;
+	}
+
+	public Command goToPosition(Rotation2d position) {
+		return new FunctionalCommand(() -> {}, () -> pivot.setPosition(position), interrupted -> pivot.stop(), () -> false, pivot)
```

Comment:
```suggestion
		return new FunctionalCommand(
		  () -> {}, () -> pivot.setPosition(position),
		  interrupted -> pivot.stop(),
		  () -> false,
		  pivot
		)
```
and formatter off too